styles.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.text {
+  width: 80%;
+  margin: 0 auto;
+  margin-top: 5em;
+}
+
+.salutation {
+  font-size: 40px;
+  font-weight: 700;
+  width: auto;
+  height: 150px;
+  color: #172b4d;
+  line-height: 52px;
+  padding-right: 20px;
+}
+
+.paragraph1 {
+  margin: 15px 0;
+  width: auto;
+}
+
+.lets-connect {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  margin-left: 0%;
+}
+
+.buttons li {
+  display: inline-block;
+  margin-right: 20px;
+  margin-top: 25px;
+}
```

Comment:
- [ ] **[REQUIRED CHANGES]** In this project we are trying to improve our skills with `flexbox`. Therefore, I would like you to change your `display` type to `flex`. Here is how you can do it:
```css
.buttons {
  display: flex;
  gap: 5px; /* gap adds a space between flex elements */
}
```

I know that flexbox can be really annoying, but it's really important that you use it, because you will need these skills further on in your curriculum. If you are having difficulties with it, please consider reading [this guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).

Reply:
Thank you for the review @fernandorpm